Langs/0415.lang

```diff
@@ -0,0 +1,58 @@
+[MetaData]
+Language=Polish - Poland
+Translator=B.O.S.S (BxOxSxS)
+Version=2.3.0.0
+[Strings]
+Donate=Wesprzyj
+Check for Updates=Sprawdź aktualizacje
+Now Reach WhyNotWin11 via https://www.whynotwin11.org/=Teraz osiągaj WhyNotWin11 przez https://www.whynotwin11.org/
+Results Based on Currently Known Requirements!=Wyniki bazujące na obecnie znanych wymaganiach!
+Translation by=Tłumaczenie od
+Architecture (CPU + OS)=Architektura (CPU + OS)
+Boot Method=Metoda Uruchamiania
+CPU Compatibility=Kompatybilność CPU
+CPU Core Count=Ilość Rdzeni CPU
+CPU Frequency=Taktowanie CPU
+DirectX + WDDM2=DirectX + WDDM2
+Disk Partition Type=Styl Partycji Dysku
+RAM Installed=Zainstalowany RAM
+Secure Boot=Bezpieczny Rozruch
```

Comment:
To nazwa wlasna to bym zostawil na secure boot

Reply:
To nieprawda Microsoft i inni producenci używają tego przetłumaczonego wyrażenia np [tutaj](https://docs.microsoft.com/pl-pl/mem/intune/user-help/you-need-to-enable-secure-boot-windows)

Reply:
O to nie wiedzialem. To moze zostac w takim razie jak jest.